src/us/kbase/auth2/lib/token/TokenType.java

```diff
@@ -16,8 +15,12 @@
 	 */
 	/** A standard login token. */
 	LOGIN				("Login", "Login"),
-	/** An extended lifetype token. */
-	EXTENDED_LIFETIME	("ExtLife", "Extended lifetime");
```

Comment:
Yeah, we'll need to dump the tokens collection. Already talked to Keith about it.